src/main/java/uk/gov/hmcts/divorce/caseworker/schedule/CaseworkerIssueAosTask.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.hmcts.divorce.caseworker.schedule;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.divorce.caseworker.service.CcdConflictException;
+import uk.gov.hmcts.divorce.caseworker.service.CcdManagementException;
+import uk.gov.hmcts.divorce.caseworker.service.CcdSearchCaseException;
+import uk.gov.hmcts.divorce.caseworker.service.CcdSearchService;
+import uk.gov.hmcts.divorce.caseworker.service.CcdUpdateService;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+
+import java.util.List;
+
+import static uk.gov.hmcts.divorce.caseworker.event.CaseworkerIssueAos.CASEWORKER_ISSUE_AOS;
+import static uk.gov.hmcts.divorce.common.model.State.Issued;
+
+@Component
+@Slf4j
+public class CaseworkerIssueAosTask {
+
+    @Autowired
+    private CcdUpdateService ccdUpdateService;
+
+    @Autowired
+    private CcdSearchService ccdSearchService;
+
+    @Scheduled(cron = "${schedule.issue_aos}")
+    public void issueAosTask() {
+
+        log.info("Issue AOS Scheduled Task started.");
+
+        try {
+            final List<CaseDetails> issuedCases = ccdSearchService.searchForAllCasesWithStateOf(Issued);
+            issuedCases.forEach(caseDetails -> {
```

Comment:
I'm not complaining but why go for .forEach over a for loop?

Reply:
There is actually no reason, probably because I autmatically typed it that way.  I would prefer it to be a for loop in this case so I will change it.